zkplus/zkplus_test.go

```diff
@@ -194,7 +195,7 @@ func TestBadConnection(t *testing.T) {
 	go func() {
 		conn = z.blockOnConn()
 	}()
-	time.Sleep(200 * time.Millisecond)
+	runtime.Gosched()
```

Comment:
Ideally there would be a wait group or equivalent inside the goroutine since reading `conn` in the assert.Nil below is a race with setting it in the goroutine (as long as it is "set" to nil there won't be a race but it is kind of a sketchy way of testing it).  Also doing `runtime.Gosched()` doesn't guarantee that that specific goroutine is going to run in the meantime.